content/zh-cn/docs/contribute/style/diagram-guide.md

```diff
@@ -580,7 +597,7 @@ First, use your external tool to create the diagram and save it as an `.svg`
 or `.png` image file. After that, use the same steps as the __Mermaid+SVG__
 method for adding `.svg` image files.
 -->
-### 外部工具
```

Comment:
Is it necessary to add the english internal link ref? The current works fine for me.

Reply:
Yes, it is required to add anchors in the zh version manually for headings 2 and 3, to keep the same anchors as en version.

Reply:
That makes sense. But what about the anchors to other headers which are different from the en anchors.

Reply:
Headings 2 and 3 are commonly referenced, while headings 4, 5, and 6 are referenced less frequently. Typically, only the former are assigned anchors manually in the process of localization.